gamemode/modules/f1menu/cl_searchbox.lua

```diff
@@ -6,15 +6,18 @@ function PANEL:Init()
     self.BaseClass.Init(self)
     self.F1Down = true
     self:SetFont("DarkRPHUD2")
-    self:SetTextColor(Color(255,255,255,255))
-    self:SetCursorColor(Color(255,255,255,255))
 
     self.lblSearch = vgui.Create("DLabel", self)
     self.lblSearch:SetFont("DarkRPHUD2")
-    self.lblSearch:SetColor(Color(200, 200, 200, 200))
     self.lblSearch:SetText(DarkRP.getPhrase("f1Search"))
     self.lblSearch:SizeToContents()
     self.lblSearch:SetPos(5)
+
+    if GAMEMODE.Config.DarkRPSkin == "DarkRP" then
+        self:SetTextColor(Color(255,255,255,255))
+        self:SetCursorColor(Color(255,255,255,255))
+        self.lblSearch:SetColor(Color(200, 200, 200, 200))
+    end
```

Comment:
> Ah, SetSkin will have not yet applied when PANEL:Init is called while ForceDermaSkin will have. It should be fine later on though so it should have no visual effect unless the control is broken and doesn't fetch the SKIN fields after PANEL:Init

Yeah, makes sense that. You see it now and again within various addon's GUIs still.

> Changes to the DarkRP skin does not affect this in any way. Baseclasses don't exist in the SKIN world.

Oh, that's good then. I don't normally work with skins at all anyway. Decided not to be daring with what was done due to that, haha. Again, more than happy to revert my change so you can instead make a pull with yours if you so wish.

> Definitely not.
Addressed in my previous comment.

Indeed:
> (also ironically goes against the whole "staying local" idea)